### PR TITLE
Adds "What's new in extensions" page

### LINKED ISSUE #316: Have a "What's New" page for Extensions
**Is your feature request related to a problem? Please describe.**
There's no single place for us (Chromium) to highlight recent changes to the platform, or for extension developers to see what's coming up in the latest releases.

**Describe the solution you'd like**
We should add a "What's new" style page for extensions, highlighting new APIs + API changes, etc.  As a first step, this can be a hand-crafted page updated periodically.  Future improvements could include automating some aspects (automatically identifying new APIs?), supporting RSS feeds, etc.

**Describe alternatives you've considered**
- A what's new across dcc.  This scope would be too large.
- Other platforms (e.g. chromium blog).  Better to centralize here.

### PR DESCRIPTION
**Pull request by awfuchs**
Added page at /docs/extensions/whatsnew

This is intended as a canonical reference for significant changes to the extensions platform, documentation, and policy.

Fixes #316

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+---
+<!--lint disable no-duplicate-headings-->
+<!--lint disable first-heading-level-->
+
+Check this page often to learn about changes to the Chrome extensions platform,
+its documentation, and related policy or other changes.
+
+### 2021.01.19: Manifest V3 launched
+
+With the launch of Chrome 88, the extensions platform now supports extensions
```
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:35]:
```suggestion
With the release of Chrome 88, the extensions platform now supports extensions
```

(Or even just "As of Chrome 88")

I don't know why, but I feel like I haven't heard "launch" applied to Chrome releases frequently.

**Review by rdcronin — APPROVED** [review]
Thanks, Andy!  We should backfill this with a few more things, too, but I think this is a perfect start.

**Review by dotproto — APPROVED** [review]
LGTM. Excited to kick off this feed 😁

**Review by dawgdemi — COMMENTED** [review]
https://chrome.google.com/webstore/detail/bitwarden-free-password-m/nngceckbapebfimnlniiiahkandclblb/reviews?hl=en

**Pull request merged**